files/en-us/web/css/@media/forced-colors/index.html

```diff
@@ -26,60 +22,81 @@ <h3 id="Values">Values</h3>
 </dl>
 
 <h2 id="Usage_notes">Usage notes</h2>
 
 <h3 id="Properties_affected_by_forced-color_mode">Properties affected by forced-color mode</h3>
 
-<p>In forced colors mode, the values of the following properties are treated as if they have no author-level values specified. That is, user-specified values (if any) or browser-specified values are used instead.</p>
+<p>In forced colors mode, the values of the following properties are treated as if they have no author-level values specified. That is, browser-specified values are used instead. The browser-specified values do not affect the style cascade; the values are instead forced by the browser at paint time. These browser-specified values are selected from the set of system colors &emdash; this ensures a consistent contrast for common UI elements for users that have forced colors enabled.</p>
 
 <ul>
  <li>{{cssxref("color")}}</li>
+ <li>{{cssxref("background-color")}}</li>
  <li>{{cssxref("text-decoration-color")}}</li>
  <li>{{cssxref("text-emphasis-color")}}</li>
  <li>{{cssxref("border-color")}}</li>
  <li>{{cssxref("outline-color")}}</li>
  <li>{{cssxref("column-rule-color")}}</li>
- <li>{{cssxref("scrollbar-color")}}</li>
  <li>{{cssxref("-webkit-tap-highlight-color")}}</li>
- <li>{{cssxref("box-shadow")}}</li>
- <li>{{cssxref("text-shadow")}}</li>
  <li><a href="/en-US/docs/Web/SVG/Attribute/fill">SVG fill attribute</a></li>
  <li><a href="/en-US/docs/Web/SVG/Attribute/stroke">SVG stroke attribute</a></li>
 </ul>
 
-<p>You can use <a href="/en-US/docs/Web/CSS/color_value#System_Colors">system color keywords</a> with any property <em>other</em> than those listed above, to ensure that the rest of the page integrates with the restricted color palette available in forced colors mode.</p>
+<p>Additionally the following properties have special behavior in forced colors mode:</p>
+<ul>
+ <li>{{cssxref("box-shadow")}} is forced to 'none'</li>
+ <li>{{cssxref("text-shadow")}} is forced to 'none'</li>
+ <li>{{cssxref("background-image")}} is forced to 'none' for values that are not url-based</li>
+ <li>{{cssxref("color-scheme")}} is forced to 'light dark'</li>
+ <li>{{cssxref("scrollbar-color")}} is forced to 'auto'</li>
+</ul>
+
+<p>The system colors that are forced for the above properties depend on the context of the element. For example the {{cssxref("color")}} property on button element will be forced to ButtonText, whereas on normal text it will be forced to CanvasText. See the list of system colors for additional details of when each might be appropriate in various UI contexts.</p>
```

Comment:
```suggestion
<p>The system colors that are forced for the above properties depend on the context of the element. For example the {{cssxref("color")}} property on button element will be forced to <code>ButtonText</code>. On normal text it will be forced to <code>CanvasText</code>. See the list of system colors for additional details of when each might be appropriate in various UI contexts.</p>
```

Shortened a sentence here as well. In addition can we link to the list of system colors if we're naming it?

Reply:
There was a previous link, but added another one here for convenience.